deploy/helm/kubernetes/values.yaml

```diff
@@ -3,3 +3,60 @@ apiServer:
   certSANs:
     dnsNames: []
     ipAddresses: []
+
+imagePullPolicy: IfNotPresent
+
+etcd:
+  image: k8s.gcr.io/etcd:3.4.3-0
```

Comment:
Please separate single image parameter to:

```yaml
etcd:
  image:
    repository: k8s.gcr.io/etcd
    tag: 3.4.3-0
```

this is more common for all helm charts. Or most full example:

```yaml
etcd:
  image:
    repository: k8s.gcr.io/etcd
    tag: 3.4.3-0
    pullPolicy: IfNotPresent
    pullSecrets:
    - name: regsecret
```